qase-javascript-commons/src/reporters/abstract-reporter.ts

```diff
@@ -110,7 +130,7 @@ export abstract class AbstractReporter implements ReporterInterface {
         this.doLogError('Caused by:', error.cause);
       }
 
-      this.logger.log(`${error.stack || `${error.name}: ${error.message}`}`);
+      this.logger.log(`${error.stack ?? `${error.name}: ${error.message}`}`);
```

Comment:
seems strange that we have 4 ` marks here. How does it work?

Reply:
I have reverted this change. If `error.stack` isn't undefined then we use it. Or use this `error.name: error.message`